bokeh/tests/test_layouts.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+import unittest
+import inspect
+
+
+def get_prop_set(class_object):
```

Comment:
I'm pretty sure this or similar method is implemented on `HasProps`, `Model` or `Viewable`.